CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 1.17.1 (Unreleased)
+
+FEATURES:
+
+* **New Argument:** `rancher2_cluster.fleet_workspace_name` - (Optional/Computed) Fleet workspace name (string)
```

Comment:
fleet workspace stuff should be removed from this PR

Reply:
@aiyengar2 , the line was there just to avoid conflicts when PR https://github.com/rancher/terraform-provider-rancher2/pull/726 was merged. Already merged.